pkg/network/telemetry/ebpf_telemetry.go

```diff
@@ -260,23 +260,21 @@ func getAllProgramSpecs(m *manager.Manager, undefinedProbes []manager.ProbeIdent
 		if err != nil {
 			return nil, err
 		}
-		if !present {
-			return nil, fmt.Errorf("could not find ProgramSpec for probe %v", p.ProbeIdentificationPair)
+		if present {
+			specs = append(specs, s...)
```

Comment:
Do we want to log a warning or error here? How common is `!present`?

Reply:
It can be very common in some scenarios. For example, for gotls prebuilt binaries will never have some probes. They will then hit `!present` everytime.

In anycase `!present` should not be an error condition here, since the ebpf-manager does not treat it as an error, i.e., it is possible to list `Probes` in the manager which are not present in the underlying ELF file.